### ISSUE TITLE
Problem having good inferences

### ISSUE THREAD

**Issue by LucBourrat1** [tjdhg456/RoomNet-Pytorch#1]
I trained the model using your code, and I used the code from the tensorflow implementation [here](https://github.com/GitBoSun/roomnet) to visualize the results of inferences on the validation set. But the results are not good... I obtain 62% accuracy for the classification and the keypoints regression is completely false, even when the model predicts the right room type.
Also during training the regression loss goes up before going down.
Can you tell me if the training was good for you and if you managed to visualize some good inferences?

**Comment by tjdhg456** [comment]
Hello.
Sorry for the inconvenience.

I didn't finish yet; I just initialized the project with some base code. (I think some parts are wrong)
I'll update the project after finishing the writing code and getting the good results.

Thank you.
